CoreWidgetProvider/Helpers/ChartHelper.cs

```diff
@@ -1,138 +1,244 @@
-                minHeight = Math.Min(minHeight, 95 - (chartValues[pointIndex] / 100 * 90));
-            }
-
-            lock (_lock)
-            {
-                g.SmoothingMode = SmoothingMode.AntiAlias;
-
-                using var darkGreyBrush = new SolidBrush(DarkGrayColor);
-                g.FillRectangle(darkGreyBrush, 0, 0, width - 1, height - 1);
-                g.DrawRectangle(Pens.LightGray, 0, 0, width - 1, height - 1);
-
-                if (chartValues.Count >= 2)
-                {
-                    if (type == "gpu")
-                    {
-                        chartPen = GPUChartPen;
-                        brush20 = GPUBrush20;
-                        brush50 = GPUBrush50;
-                        brush100 = GPUBrush100;
-                    }
-                    else if (type == "mem")
-                    {
-                        chartPen = MemChartPen;
-                        brush20 = MemBrush20;
-                        brush50 = MemBrush50;
-                        brush100 = MemBrush100;
-                    }
-                    else if (type == "net")
-                    {
-                        chartPen = NetChartPen;
-                        brush20 = NetBrush20;
-                        brush50 = NetBrush50;
-                        brush100 = NetBrush100;
-                    }
-
-                    points.Add(new PointF(startChartX, 95));
-                    points.Add(new PointF(295, 95));
-                    points.Add(new PointF(295, 95 - (chartValues.Last() / 100 * 90)));
-
-                    if (minHeight >= 77)
-                    {
-                        g.FillPolygon(brush20, points.ToArray());
-                    }
-                    else if (minHeight >= 50)
-                    {
-                        g.FillPolygon(brush50, points.ToArray());
-                    }
-                    else
-                    {
-                        g.FillPolygon(brush100, points.ToArray());
-                    }
-
-                    for (var pointIndex = 0; pointIndex < points.Count - 4; pointIndex++)
-                    {
-                        g.DrawLine(chartPen, points[pointIndex].X, points[pointIndex].Y, points[pointIndex + 1].X, points[pointIndex + 1].Y);
-                    }
-                }
-            }
-        }
-
-        var bytes = BitmapToByteArray(bitmap);
-        bitmap.Dispose();
-        points.Clear();
-        GC.Collect();
-        return bytes;
-    }
-
-    public static byte[] BitmapToByteArray(Bitmap img)
-    {
-        using var stream = new MemoryStream();
-        img.Save(stream, System.Drawing.Imaging.ImageFormat.Png);
-        return stream.ToArray();
+    public static string CreateChart(List<float> chartValues, ChartType type)
+    {
+        /* // Values to use for testing when a static image is desired.
+        chartValues.Clear();

```

Comment:
A block comment around a single-line comment?  NIT: remove //

Reply:
Or, if the comment isn't needed remove it.

Reply:
If you remove the block comment, you still want the comment explaining what it is. I suppose I could move the single line comment above the block.